src/pages/workspaces/workspace/Dashboard/CloudInformation.ts

```diff
@@ -0,0 +1,189 @@
+          text: bucketName,
+          style: { marginLeft: '0.25rem' },
+        }),
+      ]),
+      canWrite(accessLevel) &&
+        h(
+          InfoRow,
+          {
+            title: 'Estimated Storage Cost',
+            subtitle: cond(
+              [!storageCost, () => 'Loading last updated...'],
+              [
+                !!storageCost?.isSuccess,
+                () => {
+                  if (storageCost?.lastUpdated) {
+                    return `Updated on ${new Date(storageCost?.lastUpdated).toLocaleDateString()}`;
+                  }
+                  return 'Unable to determine last date updated';
+                },
+              ]
+            ),
+          },
+          [storageCost?.estimate || '$ ...']
+        ),
+      canWrite(accessLevel) &&
+        h(
+          InfoRow,
+          {
+            title: 'Bucket Size',
+            subtitle: cond(
+              [!bucketSize, () => 'Loading last updated...'],
+              [
+                !!bucketSize?.isSuccess,
+                () => {
+                  if (bucketSize?.lastUpdated) {
+                    return `Updated on ${new Date(bucketSize?.lastUpdated).toLocaleDateString()}`;
+                  }
+                  return 'Unable to determine last date updated';
+                },
+              ]
+            ),
+          },
+          [bucketSize?.usage]
+        ),
+    ]),
+    h(Fragment, [
+      div({ style: { paddingBottom: '0.5rem' } }, [
+        h(
+          Link,
+          {
+            style: { margin: '1rem 0.5rem' },
+            ...newTabLinkProps,
+            onClick: () => {
+              Ajax().Metrics.captureEvent(Events.workspaceOpenedBucketInBrowser, {
+                ...extractWorkspaceDetails(workspace),
+              });
+            },
+            href: bucketBrowserUrl(bucketName),
+          },
+          ['Open bucket in browser', icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })]
+        ),
+      ]),
+      div({ style: { paddingBottom: '0.5rem' } }, [
+        h(
+          Link,
+          {
+            style: { margin: '1rem 0.5rem' },
+            ...newTabLinkProps,
+            onClick: () => {
+              Ajax().Metrics.captureEvent(Events.workspaceOpenedProjectInConsole, {
+                ...extractWorkspaceDetails(workspace),
+              });
+            },
+            href: `https://console.cloud.google.com/welcome?project=${googleProject}`,
+          },
+          ['Open project in Google Cloud Console', icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })]
+        ),
+      ]),
+    ]),
+  ];
+};
+
+export const CloudInformation = (props: CloudInformationProps): ReactNode => {
+  const { workspace, ...rest } = props;
+  if (isAzureWorkspace(workspace)) {
+    return h(AzureCloudInformation, { workspace, ...rest });
+  }
+  if (isGoogleWorkspace(workspace)) {
+    return h(GoogleCloudInformation, { workspace, ...rest });
+  }
+  return h(Fragment);
```

Comment:
If nothing is to be rendered, suggest returning null instead of an empty fragment.

```suggestion
  return null;
```

Reply:
Habit from using Mobx ages ago.